modules/core/src/main/scala/com.snowplowanalytics.iglu/client/resolver/Resolver.scala

```diff
@@ -380,6 +439,8 @@ object Resolver {
   type SchemaListLookupResult = ResolverResult[SchemaListKey, SchemaList]
   type SupersededBy           = Option[SchemaVer.Full]
 
+  case class RawSchema(schemaKey: SchemaKey, json: Json)
```

Comment:
This case class is quite similar to a `SelfDescribingSchema[Json]`.

I think I need to see how this is used in common-streams or RDB Loader before I can comment on the return type.

Reply:
> This case class is quite similar to a SelfDescribingSchema[Json].

It's similar but there is a big difference : for `SelfDescribingSchema[Json]` the parsing of the `Json` has already happened successfully.

See how it is used in `common-streams` : https://github.com/snowplow-incubator/common-streams/blob/main/modules/loaders-common/src/main/scala/com/snowplowanalytics/snowplow/loaders/transform/SchemaProvider.scala#L31

I considered returning `SelfDescribingSchema[Json]` instead of `Json` but I wanted to leave the parsing where it currently is. I'm happy to change it if you think that it's better. I'm not very familiar with all the loaders that will cal `lookupSchemasUntil` and what the change involves.

Reply:
`SelfDescribingSchema[Json]` is the schema before parsing.  And `SelfDescribingSchema[Schema]` is the schema after parsing.  Does that change your answer at all?

Reply:
Oh sorry I missed that ! Then it's exactly what I need, perfect !